src/screenshot_gui.cpp

```diff
@@ -50,33 +48,44 @@ struct ScreenshotWindow : Window {
 	/**
 	 * Make a screenshot.
 	 * Ask for confirmation if the screenshot will be huge.
-	 * @param t Screenshot type: World, defaultzoom, heightmap or viewport screenshot
+	 * @param st Screenshot type: World, defaultzoom, heightmap or viewport screenshot
 	 */
 	static void TakeScreenshot(ScreenshotType st) {
```

Comment:
Why leave this function at all?

Reply:
A fair point. Perhaps, when making a screenshot from within the `ScreenshotWindow` GUI, one might want to override it to remove that window from the resulting screenshot.

Otherwise, it serves exactly the same purpose as the equivalent function in `toolbar_gui.cpp`.